dScripts/EquipmentTriggers/CoilBackpackBase.cpp

```diff
@@ -0,0 +1,25 @@
+#include "CoilBackpackBase.h"
+
+#include "Entity.h"
+#include "SkillComponent.h"
+
+void CoilBackpackBase::OnFactionTriggerItemEquipped(Entity* itemOwner, LWOOBJID itemObjId) {
+	itemOwner->Subscribe(itemObjId, this);
+	itemOwner->SetVar<uint8_t>(u"coilCount", 0);
+}
+
+void CoilBackpackBase::OnHitOrHealResult(Entity* self, Entity* attacker, int32_t damage) {
```

Comment:
should be `notifyHitOrHealResult`

To be consistent with the lua scripts and not conflict with a normal `OnHitOrHealResult` if there are ever two in the same script